core/state/sync.go

```diff
@@ -26,14 +26,9 @@ import (
 	"github.com/ethereumproject/go-ethereum/trie"
 )
 
-// StateSync is the main state synchronisation scheduler, which provides yet the
-// unknown state hashes to retrieve, accepts node data associated with said hashes
-// and reconstructs the state database step by step until all is done.
-type StateSync trie.TrieSync
-
 // NewStateSync create a new state trie download scheduler.
-func NewStateSync(root common.Hash, database ethdb.Database) *StateSync {
-	var syncer *trie.TrieSync
+func NewStateSync(root common.Hash, database ethdb.Database) *trie.Sync {
```

Comment:
AFAIK had been a purely vestigal unnecessary casting from an earlier set of changes, which probably should have been removed at the time of _that_ change. 

TODO(whilei): grep the history and point to relevant change. Because we want to be very sure that state sync is working properly.